packages/gestalt/src/Touchable.css

```diff
@@ -1,5 +1,8 @@
 .touchable {
   composes: accessibilityOutline from "./Focus.css";
+  display: block;
+  outline: 0;
+  transition: transform .2s ease-out;
```

Comment:
This is fine for now - we should just check in with Ashley / AJ to see what exact animation we want here. I know in the past it was something like a cubic-bezier of `0.25, 0.375, 0.1, 0.975`

The only reference I found was: https://design.pinadmin.com/pds/index.html#container-tapstate which provides more details

Reply:
this is what Duplo uses today. will check with them.

Reply:
Comparison:

`200ms ease-out`
![200-ease-out](https://user-images.githubusercontent.com/5341184/84196856-b727c380-aa55-11ea-88cc-c0f27b8d11c0.gif)

`250ms cubic-bezier(0.25, 0.375, 0.1, 0.975)`
![250ms-cubic-bezier](https://user-images.githubusercontent.com/5341184/84196863-b8f18700-aa55-11ea-9831-052c9df042b6.gif)

Reply:
Acknowledging that I've seen this but I don't have any context on this work. AJ is on PTO for the week. Is there any designer working through touch specifically?

Reply:
@bishwei do you remember why the Duplo TapArea press animation was built the way it is today?

Reply:
we based as much as we could off the ios designs.  iirc, the touch animation was done according to pds spec at that time (~2 years ago).  its definitely possible that, since then, there have been changes to the design spec or changes to TapArea.

imo we should go with the animation based on pds spec here